src/main/src/main/java/com/shevtsov/dao/impl/ClientDaoImpl.java

```diff
@@ -0,0 +1,68 @@
+package com.shevtsov.dao.impl;
+
+import com.shevtsov.dao.ClientDao;
+import com.shevtsov.domain.Client;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ClientDaoImpl implements ClientDao {
```

Comment:
Do you really need both implementations?

Reply:
no. deleted